### PR TITLE
Allow UI to initialize to old interface

### PR DESCRIPTION
**Pull request by bsteinbk**
- Add new parameter to UiFramework.initialize to allow leagacy apps to initialize UI to "deprecated" 1.0 interface.
- Add tests
- Add flag in ui-test-app that allows testing in application.

### CODE REVIEW

**Review by NancyMcCallB — APPROVED** [review]

File: ui/appui-react/src/appui-react/UiFramework.ts [hunk 1]
```diff
@@ -160,6 +162,9 @@ export class UiFramework {
     if (frameworkStateKey && store)
       UiFramework._frameworkStateKeyInStore = frameworkStateKey;
 
+    if (startInUi1Mode)
+      UiFramework.store.dispatch({ type: ConfigurableUiActionId.SetFrameworkVersion, payload: "1" });
```
Comment (aruniverse) [inline ui/appui-react/src/appui-react/UiFramework.ts:166]: Why was this added to core and not something DR could call after initialing UiFramework themselves?

Comment (bsteinbk) [reply ui/appui-react/src/appui-react/UiFramework.ts:166]: @aruniverse - We saw strange crashes in UI components if this was done after the initalize promise resolved.  I did not see any errors if the update was done during initialize processing. You can make the change once the entire frontstage is rendered without issue.

**Comment by bsteinbk** [comment]
@mergifyio backport release/3.1.x

**Comment by calebmshafer** [comment]
@bsteinbk any reason to not backport this to 3.0.x in addition to 3.1.x?

**Comment by bsteinbk** [comment]
@calebmshafer I don't think we need to. From what I heard DR will be moving to 3.1 or 3.2 before they release and this is only needed by those types of legacy apps that need to allow UI 1.0 mode for a short transition period. We don't want to endorse other apps to support the old UI.
